source/_css/components/_postShorten.scss

```diff
@@ -17,6 +17,10 @@
 
     .postShorten-thumbnailimg {
         overflow:hidden;
+
+        img{
+           border-radius: 5px;
```

Comment:
Let's remove the border-radius rule to be consistent with the other type of thumbnails.